buildSrc/src/main/kotlin/io/spine/internal/gradle/publish/CloudRepo.kt

```diff
@@ -26,9 +26,38 @@
 
 package io.spine.internal.gradle.publish
 
+import io.spine.internal.gradle.Repository
+
 /**
- * Popular names of Gradle configurations.
+ * CloudRepo Maven repository.
+ *
+ * There is a special treatment for this repository. Usually, fetching and pushing of artifacts
+ * is performed via the same URL. But it is not true for CloudRepo. Fetching is performed via
+ * public repository, and pushing via private one. Their URLs differ in `/public` infix.
  */
-internal object ConfigurationName {
-    const val archives = "archives"
+internal object CloudRepo {
+
+    private const val name = "CloudRepo"
+    private const val baseUrl = "https://spine.mycloudrepo.io/public/repositories"
+    private const val credentialsFile = "cloudrepo.properties"
+
+    /**
+     * Use this instance to depend on artifacts from this repository.
+     */
+    val reading = Repository(
+        name = name,
+        releases = "$baseUrl/releases",
+        snapshots = "$baseUrl/snapshots",
+        credentialsFile = credentialsFile
+    )
+
+    /**
+     * Use this instance to publish artifacts to this repository.
+     */
+    val writing = Repository(
```

Comment:
and here I'd say it is `CloudRepo.destination`. Besides, it's only used once at the moment from what I can see.